platform/core/src/types/HangingProtocol.ts

```diff
@@ -1,63 +1,86 @@
-type DisplaySetInfo = {
-  SeriesInstanceUID: string;
-  displaySetInstanceUID: string;
-  displaySetOptions: Record<string, unknown>;
+export type DisplaySetInfo = {
+  displaySetInstanceUID?: string;
```

Comment:
why removing the SeriesInstanceUID from here and also from `DisplaySetMatchDetails`? `DisplaySetMatchDetails` even has `StudyInstanceUID`

Reply:
Because display sets may not be a single study or series.    If you want to match on that attribute, you should grab it from the definitive source, which is the set of instances in the display set, just like every other attribute.  Same as your comment above about getting the hpInfo from the hanging protocol service - it shouldn't get replicated if there is a definitive location to grab it from.